libraries/resource_sensu_metric_dirsize.rb

```diff
@@ -0,0 +1,20 @@
+require "chef/resource"
+
+class Chef
+  class Resource
+    class SensuPluginMetricDirsize < Chef::Resource::SensuPluginMetricGeneric
+
+      def initialize(name, run_context=nil)
+        super
+        @cmd_name = "metric-dirsize.rb"
+        @remote_files << {name: @cmd_name, source: "https://raw.githubusercontent.com/sensu/sensu-community-plugins/master/plugins/files/metric-dirsize.rb"}
+      end
+
+      set_attrib :directory, :cmd_flag => '-d', :kind_of => String, :required => true
+      set_attrib :directory, :cmd_flag => '-r'
```

Comment:
The attribute name `directory` has to be changed accordingly here and in the two lines below.